APMrover2/GCS_Mavlink.cpp

```diff
@@ -175,6 +175,19 @@ void GCS_MAVLINK_Rover::send_rangefinder() const
         voltage);
 }
 
+/*
+  send RPM packet
+ */
+void NOINLINE Rover::send_rpm(mavlink_channel_t chan)
```

Comment:
Note that this message was previously being used to send the encoder values. There was talk about using a different message for the encoders, but not sure where that is currently at.